src/event-bus.ts

```diff
@@ -166,24 +161,19 @@ export class EventBus<EventBase extends IEvent = IEvent>
         filter((e) => !!e),
         mergeMap((command) =>
           defer(() => this.commandBus.execute(command)).pipe(
-            catchError((error) =>
-              throwError(() => this.mapToUnhandledErrorInfo(command, error)),
-            ),
+            catchError((error) => {
+              const unhandledError = this.mapToUnhandledErrorInfo(event, error);
```

Comment:
should `this.mapToUnhandledErrorInfo(event, error);` this be `this.mapToUnhandledErrorInfo(command, error);`

`event` variable  is not defined in this scope I believe. Because of that whenever something fails in command bus it throws internal error like following:
`ReferenceError: event is not defined`

Reply:
version 10.0.2 crashes app with this error

Reply:
Fixed in 10.0.3